images/macos/provision/configuration/configure-machine.sh

```diff
@@ -17,6 +17,11 @@ if [ -d "/Library/Application Support/VMware Tools" ]; then
     sudo "/Library/Application Support/VMware Tools/vmware-resolutionSet" 1176 885
 fi
 
+# Update VoiceOver Utility to allow VoiceOver to be controlled with AppleScript by creating a special file (SIP must be disabled)
+if csrutil status | grep -Eq  "System Integrity Protection status: (disabled|unknown)"; then
+    sudo touch /private/var/db/Accessibility/.VoiceOverAppleScriptEnabled
```

Comment:
Consider:

```suggestion
    sudo echo -n "a" > /private/var/db/Accessibility/.VoiceOverAppleScriptEnabled
```

To more closely mimic the system behaviour.

Reply:
Interestingly, when I enabled the setting via system menu this file didn't contain anything

Reply:
Curious! Perhaps a subtle difference between MacOS version? This is on Monterey:

![Screenshot 2021-12-25 at 08 26 38](https://user-images.githubusercontent.com/11313985/147381669-5b4090c6-60db-421d-a584-8aacd2694780.png)

![Screenshot 2021-12-25 at 08 28 18](https://user-images.githubusercontent.com/11313985/147381671-3403935d-d419-4e0c-8832-2e068948aafe.png)

![Screenshot 2021-12-25 at 09 02 10](https://user-images.githubusercontent.com/11313985/147381672-25e967a6-3f08-4ef2-a494-77dd2f882a33.png)

But not one to worry about today - Merry Christmas! ⛄

Reply:
Indeed, this is true for Big Sur as well! I must have overlooked it. Thanks for the help!